app/main/spawn.js

```diff
@@ -12,13 +12,21 @@ specific language governing permissions and limitations under the License.
 
 import { spawn } from 'child_process';
 import path from 'path';
+import * as endpoints from '../shared/constants/endpoints';
 
-const UA_SERVICE_PATH = path.join(__dirname, '..', 'services', 'user-agent-service', 'index');
+const UA_SERVICE_BIN = path.join(__dirname, '..', 'services',
+                       'user-agent-service', 'bin', 'user-agent-service');
+const DB_PATH = path.join(__dirname, '..', '..');
 
 // @TODO Use something like `forever` or `pm2` to ensure that this
 // process remains running
+// @TODO Will `node` be accessible via path on users machines that don't
+// have node installed as an engineer?
 export function userAgentService() {
-  spawn('node', [UA_SERVICE_PATH], {
+  spawn('node', [UA_SERVICE_BIN,
```

Comment:
In order to get an ES6 repl here, we need to use `babel-node`.

On the plus side, `babel-node` will be in `node_modules`, and takes care of finding the environment `node` for us. Perhaps use that here?